src/platform_impl/ios/window.rs

```diff
@@ -504,6 +537,8 @@ impl Default for PlatformSpecificWindowBuilderAttributes {
             root_view_class: class!(UIView),
             hidpi_factor: None,
             valid_orientations: Default::default(),
+            prefers_home_indicator_hidden: false,
+            preferred_screen_edges_deferring_system_gestures: ScreenEdge::NONE,
```

Comment:
It would be cute to impl `Default` on `ScreenEdge` to codify that this is the default value (https://docs.rs/bitflags/1.1.0/bitflags/#default)